genestack_client/genestack_connection.py

```diff
@@ -127,7 +128,8 @@ def login(self, email, password):
         self.check_version()
         logged = self.application('genestack/signin').invoke('authenticate', email, password)
         if not logged['authenticated']:
-            raise GenestackAuthenticationException("Fail to login %s" % email)
+            hostname = urlsplit(self.server_url).hostname
+            raise GenestackAuthenticationException("Fail to login with %s to %s" % (email, hostname))
```

Comment:
`http://platform.genestack.org/endpoint/`   does not look good for me

Reply:
I think we should leave full URL here, so that it would fully match what is saved in config file.  In theory, there could be several frontend instances working on one host even.

Reply:
> I think we should leave full URL here, so that it would fully match what is saved in config file.

We store host in config file.